actions/setup/action.yml

```diff
@@ -21,10 +21,6 @@ inputs:
     description: Number of commits to fetch. 0 indicates all history for all branches and tags.
     default: 0
 
-  git-branch:
-    description: What branch to checkout
-    default: master
```

Comment:
Not sure if this is backwards compatible, perhaps only remove the usage of it below?

Reply:
Let's hope this hasn't been used widely yet, and deal with any breakage 🙏